srcs/jobs/jobs_fg.c

```diff
@@ -0,0 +1,83 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   jobs_fg.c                                          :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: rkuijper <rkuijper@student.codam.nl>         +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2019/10/18 17:12:11 by rkuijper       #+#    #+#                */
+/*   Updated: 2019/10/30 13:35:11 by rkuijper      ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "vsh.h"
+#include <term.h>
+#include <errno.h>
+#include <signal.h>
+#include <stdio.h>
+
+static t_proc	*last_proc(t_job *job)
+{
+	t_proc *proc;
+
+	proc = job->processes;
+	while (proc->next != NULL)
+		proc = proc->next;
+	return (proc);
+}
+
+static int		job_stop(t_job *job)
+{
+	jobs_add_job(g_data, job);
+	if (g_state->shell_type == SHELL_INTERACT)
+		ft_printf("\r\x1b[0K^Z\n");
+	job->bg = true;
+	jobs_print_job_info(job, JOB_OPT_L, g_data->jobs->joblist);
+	return (146);
```

Comment:
🤔

Reply:
Yes, this will get a comment in the near future.